src/js/outreach/apps/opt-in_app.js

```diff
@@ -11,30 +9,28 @@ import {
 
 import { DialogView } from 'js/outreach/views/dialog_views';
 
-const StateModel = Backbone.Model.extend({
-  defaults: {
-    firstName: '',
-    lastName: '',
-    dob: '',
-    phone: '',
+export default App.extend({
+  beforeStart() {
+    return Radio.request('entities', 'outreach:model', {
+      first_name: '',
+      last_name: '',
+      birth_date: '',
+      phone: '',
+    });
   },
-});
+  onStart(options, outreach) {
+    this.outreach = outreach;
 
-export default App.extend({
-  StateModel,
-  onStart() {
     const dialogView = new DialogView();
     this.showView(dialogView);
 
     this.showOptInView();
   },
   showOptInView() {
-    const optInView = new OptInView({ model: this.getState() });
+    const optInView = new OptInView({ model: this.outreach });
 
-    this.listenTo(optInView, 'click:submit', () => {
-      optInPostRequest({
-        inputData: this.getState(),
-      })
+    this.listenTo(optInView, 'click:submit', ({ model }) => {
+      this.outreach.registerPatient(model.attributes)
```

Comment:
I think this'd be something like `Radio.request('entities', 'register:patient', model.attributes);` and the model can go back to being a state model.  This just isn't an outreach model.